sqlstore/src/main/resources/db/migration/V8501__dilution_changelog.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE DilutionChangeLog (
+  dilutionChangeLogId bigint(20) NOT NULL AUTO_INCREMENT,
+  dilutionId bigint(20) NOT NULL,
+  columnsChanged varchar(500) NOT NULL,
+  userId bigint(20) NOT NULL,
+  message longtext NOT NULL,
+  changeTime TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP(),
+  PRIMARY KEY (dilutionChangeLogId),
+  CONSTRAINT fk_dilutionChangeLog_dilution FOREIGN KEY (dilutionId) REFERENCES LibraryDilution(dilutionId),
+  CONSTRAINT fk_dilutionChangeLog_user FOREIGN KEY (userId) REFERENCES User(userId)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO DilutionChangeLog (dilutionId, columnsChanged, userId, message, changeTime)
+SELECT dilutionId, columnsChanged, userId, REPLACE(message, CONCAT(d.name, ' '), ''), changeTime
+FROM LibraryDilution d
+JOIN Library l ON l.libraryId = d.library_libraryId
+JOIN LibraryChangeLog c ON c.libraryId = l.libraryId
+  AND c.message = CONCAT('Library Dilution ', d.name, ' created.');
+
+INSERT INTO DilutionChangeLog (dilutionId, columnsChanged, userId, message, changeTime)
```

Comment:
I'm sure it's fine, but also moderately terrifying to do this.

Reply:
Yep!